include/miopen/miopen.h

```diff
@@ -6155,6 +6155,71 @@ MIOPEN_EXPORT miopenStatus_t miopenT5LayerNormBackward(miopenHandle_t handle,
 // CLOSEOUT LAYERNORM DOXYGEN GROUP
 #endif
 
+#ifdef MIOPEN_BETA_API
+// Outer APIs
+/** @addtogroup outer
+ *
+ *  @{
+ */
+/*! @brief Execute a outer forward layer
+ *
+ * @param handle         MIOpen handle (input)
+ * @param x1Desc         Tensor descriptor of input tensor x1 (input)
+ * @param x1             Source data tensor x1 (input)
+ * @param x2Desc         Tensor descriptor of input tensor x2 (input)
+ * @param x2             Source data tensor x2 (input)
+ * @param yDesc          Tensor descriptor of output tensor y (input)
+ * @param y              Data tensor y (output)
+ * @return               miopenStatus_t
+ */
+MIOPEN_EXPORT miopenStatus_t miopenOuterForward(miopenHandle_t handle,
+                                                const miopenTensorDescriptor_t x1Desc,
+                                                const void* x1,
+                                                const miopenTensorDescriptor_t x2Desc,
+                                                const void* x2,
+                                                const miopenTensorDescriptor_t yDesc,
+                                                void* y);
+/*! @brief Execute a outer backwardGrad1 layer
+ *
+ * @param handle         MIOpen handle (input)
+ * @param x2Desc         Tensor descriptor of input tensor x2 (input)
+ * @param x2             Source data tensor x2 (input)
+ * @param x1GradDesc     Tensor descriptor of input tensor x1Grad (input)
```

Comment:
And x.grad is output in bwd. Fix both comments from bwd ops.

Reply:
I replaced.